docs/content/en/docs/yaml-crd-ref/app.md

```diff
@@ -128,12 +139,40 @@ spec:
   - my-prometheus-definition
 ```
 
+### Referencing a task in a customized application container
+
+For an example of a `KeptnApp` resource definition
+that references a customized application container, see
```

Comment:
since this part is how to refer to a keptnTaskDefinition, is it worth repeating it? I wonder if we could have a single section for task definition, followed by all 3 current possibilities ?

Reply:
Which three current possibilities do you mean?  Deno-runtime has three possibilities, then is python-runtime separate and generic container yet another case? 

And yes, I think we definitely need more container examples here!  I went with what I had ;-)  And we probably need better descriptions of the examples.

Reply:
And see comment above -- I got confused by that `container-task` in the example...  Will fix this.  In point of fact, they could have a task defined as a Deno-script and then change that task to call a container and, if they keep the task name the same, they don't need to touch KeptnApp, do they?

Reply:
pretty much yes you change the spec part of the task definition from function: blabla to  python: python stuff and this works